yellowbrick/cluster/elbow.py

```diff
@@ -260,7 +293,25 @@ def fit(self, X, y=None, **kwargs):
             self.k_timers_.append(time.time() - start)
             self.k_scores_.append(
                 self.scoring_metric(X, self.estimator.labels_)
-            )
+            )    
+
+        if self.locate_elbow:
+            if self.metric == 'distortion':
+                self.curve_nature = 'convex'
+                self.curve_direction = 'decreasing'
+            elif self.metric=='silhouette' or self.metric=='calinski_harabaz':
+                self.curve_nature = 'concave'
+                self.curve_direction = 'increasing'    
+            self.elbow_locator = KneeLocator(self.k_values_,self.k_scores_,curve_nature=self.curve_nature,curve_direction=self.curve_direction)
```

Comment:
I thought we agreed that this would be a local variable and not stored on the class? I still think that's the right way to go.